wagtailio/project_styleguide/templates/patterns/includes/footer.html

```diff
@@ -1,3 +1,70 @@
 <footer class="footer">
     {% include "patterns/includes/menu_footer.html" %}
+    <div class="footer__base">
+        <div class="grid">
+            {# Links #}
+            <ul class="footer__links-wrap">
+                <li class="footer__link-item">
+                    <a class="footer__icon-link footer__icon-link--sponsor" href="/sponsors" aria-label="Sponsor Wagtail">
+                        <svg class="footer__icon" aria-hidden="true">
+                            <use xlink:href="#money"></use>
+                        </svg>
+                        <div class="footer__icon-link-content">
+                            <p class="footer__icon-link-heading">Sponsor Wagtail</p>
+                            <p class="mini-meta">Help build Wagtail enhancements and support the long-term health of the project</p>
+                        </div>
+                    </a>
+                </li>
+                <li class="footer__link-item">
+                    <a class="footer__icon-link footer__icon-link--contribute" href="https://docs.wagtail.org/en/stable/contributing/index.html#contributing-to-wagtail" aria-label="Contribute code">
+                        <svg class="footer__icon" aria-hidden="true">
+                            <use xlink:href="#code-file"></use>
+                        </svg>
+                        <div class="footer__icon-link-content">
+                            <p class="footer__icon-link-heading">Contribute code</p>
+                            <p class="mini-meta">200+ contributors in 2022!</p>
+                        </div>
+                    </a>
+                </li>
+            </ul>
+
+            <div class="footer__sign-up">
+                {# Form sign up #}
+                <p class="teaser-heading">This Week in Wagtail</p>
+
+                {% comment %}TODO - add sign up form here once created{% endcomment %}
+
+                {# Socials #}
+                <div class="footer__socials">
+                    <p class="footer__social-heading heading-four">Follow us</p>
+                    <ul class="footer__social-list">
+                        <li>
+                            <a href="https://github.com/wagtail/wagtail" aria-label="Wagtail on Github">
```

Comment:
Note: I guess this is an okay link, it feels a bit weird to link here instead of Github in my opinion.

Reply:
> instead of Github

Whereabouts do you mean? https://github.com/wagtail/ ?